dingo/gw/noise/utils.py

```diff
@@ -0,0 +1,245 @@
+import argparse
+import os.path
+import pickle
+from io import StringIO
+from os.path import join
+from os.path import isfile
+import glob
+import copy
+
+import numpy as np
+import requests
+import yaml
+
+from gwpy.table import EventTable
+from dingo.gw.noise.asd_dataset import ASDDataset
+
+"""
+Catalogue against which to check that no event is present in the estimated PSDs
+"""
+CATALOGUEs = ["GWTC-1-confident", "GWTC-2.1-confident", "GWTC-3-confident"]
```

Comment:
It's a bit awkward with the small "s", and probably better to use the American spelling, CATALOGS.